dcspy/models.py

```diff
@@ -317,7 +321,7 @@ def depiction(self) -> ControlDepiction:
         """
         Return the depiction of the control.
 
-        :return: ControlDepiction object representing the control's name, description and physical variant.
+        :return: ControlDepiction object representing the control's name amd description.
```

Comment:
Unrelated change, just removed unused text as the physical variant has now been removed.